build_tools/bazel/build_core.sh

```diff
@@ -69,7 +69,7 @@ fi
 # `bazel test //...` because the latter excludes targets tagged "manual". The
 # "manual" tag allows targets to be excluded from human wildcard builds, but we
 # want them built by CI unless they are excluded with "nokokoro".
-bazel query //iree/... | \
+bazel query //iree/... + //bindings/... | \
```

Comment:
Add a TODO noting that this will be separated out later?

Reply:
Good idea!